mappings/net/minecraft/village/VillagerProfession.mapping

```diff
@@ -1,5 +1,51 @@
 CLASS net/minecraft/unmapped/C_yhqxfbuc net/minecraft/village/VillagerProfession
 	FIELD f_acjlkvad secondaryJobSites Lcom/google/common/collect/ImmutableSet;
-	FIELD f_ayvuterw gatherableItems Lcom/google/common/collect/ImmutableSet;
+	FIELD f_ayvuterw requestedItems Lcom/google/common/collect/ImmutableSet;
+	FIELD f_kmphlego acquirableJobSite Ljava/util/function/Predicate;
+	FIELD f_obosegzt heldJobSite Ljava/util/function/Predicate;
```

Comment:
Is this the job site currently held? If so I think `currentJobSite` would be better?

Reply:
professions have no notion of whether or not the villager remembers a specific job site, that's up to their brain. This is just a predicate that matches whether the POI they're looking at is a valid job site for them. I think I know some better mappings actually, `jobSite` and `potentialJobSite` as propagated from the `MemoryModuleType` they are paired with in `VillagerLaskListProvider#createCoreTasks`